src/cloud-api-adaptor/docker/README.md

```diff
@@ -161,3 +235,19 @@ For debugging you can use docker commands like `docker ps`, `docker logs`, `dock
 kubectl delete deployment nginx
 ```
 
+## Troubleshooting
+
+When using `containerd` and `nydus-snapshotter` you might encounter pod creation failure due to
+issues with unpacking of image. Check the `nydus-snapshotter` troubleshooting [doc](../docs/troubleshooting/nydus-snapshotter.md).
+
+In order to login to the worker node you can use either of the following approaches
+
+```sh
+kubectl debug node/peer-pods-worker -it --image=busybox
```

Comment:
Maybe user error, but when I do this I can't access ctr, I guess because we are running in the debug container?
```
# kubectl debug node/peer-pods-worker -it --image=busybox
Creating debugging pod node-debugger-peer-pods-worker-n7rlf with container debugger on node peer-pods-worker.
If you don't see a command prompt, try pressing enter.
/ #
/ # ctr -n k8s.io content fetch quay.io/prometheus/busybox:latest
sh: ctr: not found
```

Reply:
Yes.. you'll need to execute `chroot /host` to get the the node fs.

Reply:
Yeah, sorry - I'm rushing too much for my brain to kick in enough

Reply:
np, I have also added it the doc